content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/image.html

```diff
@@ -16,6 +16,7 @@
 <div data-sly-use.image="com.adobe.cq.wcm.core.components.models.Image"
      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
      data-sly-test="${image.src}"
+     data-cq-asset-id="${image.uuid}"
```

Comment:
can't we finally get rid of the term "cq"?

Reply:
Thanks @kwin - I agree, we already have `data-asset`, `data-title` so I would continue with `data-asset-id`, I don't see a real win with a namespace. 

I would also move this attribute below `data-asset`.